consumer/pdata/generated_metrics.go

```diff
@@ -607,8 +607,6 @@ func (ms Metric) SetUnit(v string) {
 	(*ms.orig).Unit = v
 }
 
-
```

Comment:
`make fmt` seems to touch this file. I can revert if needed.

Reply:
It shouldn't anymore.